lib/build.js

```diff
@@ -135,27 +137,63 @@ module.exports = {
     return value;
   },
 
+  errorParse: function() {
+    this.match = [];
+    var regex = XRegExp(this.cmd.errorMatch);
```

Comment:
Depending on the regex used the multiline flag possibly needs to be added. But with quick and dirty regex for g++ it is not needed.